<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="height=device-height, width=device-width, initial-scale=1.0, minimum-scale=1.0">
    <meta name="generator" content="Hugo 0.124.1">
    <meta name="generator" content="Relearn 5.3.1">
    <meta name="description" content="Documentation and tutorials for Captus">
    <meta name="author" content="Edgardo M. Ortiz">
    <title>Output Files :: Captus documentation</title>
    <link href="/captus.docs/images/favicon.ico?1729274442" rel="icon" type="image/x-icon">
    <!-- https://github.com/filamentgroup/loadCSS/blob/master/README.md#how-to-use -->
    <link href="/captus.docs/css/fontawesome-all.min.css?1729274442" rel="stylesheet" media="print" onload="this.media='all';this.onload=null;"><noscript><link href="/captus.docs/css/fontawesome-all.min.css?1729274442" rel="stylesheet"></noscript>
    <link href="/captus.docs/css/featherlight.min.css?1729274442" rel="stylesheet" media="print" onload="this.media='all';this.onload=null;"><noscript><link href="/captus.docs/css/featherlight.min.css?1729274442" rel="stylesheet"></noscript>
    <link href="/captus.docs/css/auto-complete.css?1729274442" rel="stylesheet" media="print" onload="this.media='all';this.onload=null;"><noscript><link href="/captus.docs/css/auto-complete.css?1729274442" rel="stylesheet"></noscript>
    <link href="/captus.docs/css/perfect-scrollbar.min.css?1729274442" rel="stylesheet">
    <link href="/captus.docs/css/nucleus.css?1729274442" rel="stylesheet">
    <link href="/captus.docs/css/fonts.css?1729274442" rel="stylesheet" media="print" onload="this.media='all';this.onload=null;"><noscript><link href="/captus.docs/css/fonts.css?1729274442" rel="stylesheet"></noscript>
    <link href="/captus.docs/css/theme.css?1729274442" rel="stylesheet">
    <link href="/captus.docs/css/theme-green.css?1729274442" rel="stylesheet" id="variant-style">
    <link href="/captus.docs/css/ie.css?1729274442" rel="stylesheet">
    <link href="/captus.docs/css/variant.css?1729274442" rel="stylesheet">
    <link href="/captus.docs/css/print.css?1729274442" rel="stylesheet" media="print">
    <script src="/captus.docs/js/variant.js?1729274442"></script>
    <script>
      // hack to let hugo tell us how to get to the root when using relativeURLs, it needs to be called *url= for it to do its magic:
      // https://github.com/gohugoio/hugo/blob/145b3fcce35fbac25c7033c91c1b7ae6d1179da8/transform/urlreplacers/absurlreplacer.go#L72
      var index_url="/captus.docs/index.json";
      var root_url="/";
      var baseUri=root_url.replace(/\/$/, '');
      // translations
      window.T_Copy_to_clipboard = 'Copy to clipboard';
      window.T_Copied_to_clipboard = 'Copied to clipboard!';
      window.T_Copy_link_to_clipboard = 'Copy link to clipboard';
      window.T_Link_copied_to_clipboard = 'Copied link to clipboard!';
      // some further base stuff
      var baseUriFull='https:\/\/edgardomortiz.github.io\/captus.docs/';
      window.variants && variants.init( [ 'green' ] );
    </script>
    <script src="/captus.docs/js/jquery.min.js?1729274442" defer></script>


  </head>
  <body class="mobile-support default disableInlineCopyToClipboard" data-url="/captus.docs/assembly/extract/output/">
    <div id="body" class="default-animation">
      <div id="sidebar-overlay"></div>
      <div id="toc-overlay"></div>
      <nav id="topbar" class="highlightable">
        <div>
          <div class="navigation">
             <a class="nav nav-next" href="/captus.docs/assembly/extract/report/" title="HTML Report (&#129106;)"><i class="fas fa-chevron-right fa-fw"></i></a>
          </div>
          <div class="navigation">
             <a class="nav nav-prev" href="/captus.docs/assembly/extract/options/" title="Options (&#129104;)"><i class="fas fa-chevron-left fa-fw"></i></a>
          </div>
          <div id="top-github-link">
            <a class="github-link" title='Edit (CTRL+ALT+e)' href="https://github.com/edgardomortiz/Captus/tree/master/docs/content/assembly/extract/output.md" target="blank">
              <i class="fas fa-pen fa-fw"></i>
            </a>
          </div>
          <div id="breadcrumbs">
            <span id="sidebar-toggle-span">
              <a href="#" id="sidebar-toggle" title='Menu (CTRL+ALT+m)'><i class="fas fa-bars fa-fw"></i></a>
            </span>
            <span id="toc-menu" title='Table of Contents (CTRL+ALT+t)'><i class="fas fa-list-alt fa-fw"></i></span>
            <ol class="links" itemscope itemtype="http://schema.org/BreadcrumbList">
              <li itemscope itemtype="https://schema.org/ListItem" itemprop="itemListElement"><a itemprop="item" href="/captus.docs/"><span itemprop="name">Captus</span></a><meta itemprop="position" content="1"> > </li>
              <li itemscope itemtype="https://schema.org/ListItem" itemprop="itemListElement"><a itemprop="item" href="/captus.docs/assembly/"><span itemprop="name">Assembly</span></a><meta itemprop="position" content="2"> > </li>
              <li itemscope itemtype="https://schema.org/ListItem" itemprop="itemListElement"><a itemprop="item" href="/captus.docs/assembly/extract/"><span itemprop="name">Extract</span></a><meta itemprop="position" content="3"> > </li>
              <li itemscope itemtype="https://schema.org/ListItem" itemprop="itemListElement"><span itemprop="name">Output Files</span><meta itemprop="position" content="4"></li>
            </ol>
          </div>
          <div class="default-animation progress">
            <div class="wrapper">
<nav id="TableOfContents">
  <ul>
    <li>
      <ul>
        <li><a href="#1-sample_name__captus-ext">1. <strong><code>[SAMPLE_NAME]__captus-ext</code></strong></a></li>
        <li><a href="#2-01_coding_nuc-02_coding_ptd-03_coding_mit">2. <strong><code>01_coding_NUC</code></strong>, <strong><code>02_coding_PTD</code></strong>, <strong><code>03_coding_MIT</code></strong></a></li>
        <li><a href="#3-marker_type_coding_aafaa-01_aa">3. <strong><code>[MARKER_TYPE]_coding_AA.faa</code></strong>, <strong><code>01_AA</code></strong></a></li>
        <li><a href="#4-marker_type_coding_ntfna-02_nt">4. <strong><code>[MARKER_TYPE]_coding_NT.fna</code></strong>, <strong><code>02_NT</code></strong></a></li>
        <li><a href="#5-marker_type_genesfna-03_genes">5. <strong><code>[MARKER_TYPE]_genes.fna</code></strong>, <strong><code>03_genes</code></strong></a></li>
        <li><a href="#6-marker_type_genes_flankedfna-04_genes_flanked">6. <strong><code>[MARKER_TYPE]_genes_flanked.fna</code></strong>, <strong><code>04_genes_flanked</code></strong></a></li>
        <li><a href="#7-marker_type_contigs_listtxt">7. <strong><code>[MARKER_TYPE]_contigs_list.txt</code></strong></a></li>
        <li><a href="#8-marker_type_contigsgff">8. <strong><code>[MARKER_TYPE]_contigs.gff</code></strong></a></li>
        <li><a href="#9-marker_type_recovery_statstsv">9. <strong><code>[MARKER_TYPE]_recovery_stats.tsv</code></strong></a></li>
        <li><a href="#10-marker_type_scipio_finallog">10. <strong><code>[MARKER_TYPE]_scipio_final.log</code></strong></a></li>
        <li><a href="#11-00_initial_scipio_marker_type">11. <strong><code>00_initial_scipio_[MARKER_TYPE]</code></strong></a></li>
        <li><a href="#12-04_misc_dna-05_clusters">12. <strong><code>04_misc_DNA</code></strong>, <strong><code>05_clusters</code></strong></a></li>
        <li><a href="#13-marker_type_matchesfna-01_matches">13. <strong><code>[MARKER_TYPE]_matches.fna</code></strong>, <strong><code>01_matches</code></strong></a></li>
        <li><a href="#14-marker_type_matches_flankedfna-02_matches_flanked">14. <strong><code>[MARKER_TYPE]_matches_flanked.fna</code></strong>, <strong><code>02_matches_flanked</code></strong></a></li>
        <li><a href="#15-marker_type_contigs_listtxt">15. <strong><code>[MARKER_TYPE]_contigs_list.txt</code></strong></a></li>
        <li><a href="#16-marker_type_contigsgff">16. <strong><code>[MARKER_TYPE]_contigs.gff</code></strong></a></li>
        <li><a href="#17-marker_type_recovery_statstsv">17. <strong><code>[MARKER_TYPE]_recovery_stats.tsv</code></strong></a></li>
        <li><a href="#18-marker_type_blat_searchlog">18. <strong><code>[MARKER_TYPE]_blat_search.log</code></strong></a></li>
        <li><a href="#19-06_assembly_annotated">19. <strong><code>06_assembly_annotated</code></strong></a></li>
        <li><a href="#20-sample_name_hit_contigsfasta">20. <strong><code>[SAMPLE_NAME]_hit_contigs.fasta</code></strong></a></li>
        <li><a href="#21-sample_name_hit_contigsgff">21. <strong><code>[SAMPLE_NAME]_hit_contigs.gff</code></strong></a></li>
        <li><a href="#22-sample_name_recovery_statstsv">22. <strong><code>[SAMPLE_NAME]_recovery_stats.tsv</code></strong></a></li>
        <li><a href="#23-leftover_contigsfastagz">23. <strong><code>leftover_contigs.fasta.gz</code></strong></a></li>
        <li><a href="#24-leftover_contigs_after_custeringfastagz">24. <strong><code>leftover_contigs_after_custering.fasta.gz</code></strong></a></li>
        <li><a href="#25-captus-assembly_extractrefsjson">25. <strong><code>captus-assembly_extract.refs.json</code></strong></a></li>
        <li><a href="#26-captus-assembly_extractstatstsv">26. <strong><code>captus-assembly_extract.stats.tsv</code></strong></a></li>
        <li><a href="#27-captus-assembly_extractreporthtml">27. <strong><code>captus-assembly_extract.report.html</code></strong></a></li>
        <li><a href="#28-captus-assembly_extractlog">28. <strong><code>captus-assembly_extract.log</code></strong></a></li>
        <li><a href="#29-clust_id_cov_captus_clusters_refsfasta">29. <strong><code>clust_id##.##_cov##.##_captus_clusters_refs.fasta</code></strong></a></li>
        <li><a href="#fasta-headers-explanation">FASTA headers explanation</a></li>
      </ul>
    </li>
  </ul>
</nav>
            </div>
          </div>
        </div>
      </nav>
      <main id="body-inner" class="highlightable default" tabindex="-1">
        <div class="flex-block-wrapper">
          <div id="head-tags">
          </div>
          <article class="default">
<h1>Output Files</h1>

<p>For this example we will use the directory <code>02_assemblies</code> previously created with the <a href="https://edgardomortiz.github.io/captus.docs/assembly/assemble/output/"><code>assemble</code> module</a>. We run the following <code>Captus</code> command to search and extract our reference markers from the assemblies:</p>
<div class="highlight"><pre tabindex="0" style="color:#f8f8f2;background-color:#272822;-moz-tab-size:4;-o-tab-size:4;tab-size:4;"><code class="language-console" data-lang="console"><span style="display:flex;"><span>captus_assembly extract -a 02_assemblies \
</span></span><span style="display:flex;"><span>-n Angiosperms353 \
</span></span><span style="display:flex;"><span>-p SeedPlantsPTD \
</span></span><span style="display:flex;"><span>-m SeedPlantsMIT \
</span></span><span style="display:flex;"><span>-d noncoding_DNA.fasta \
</span></span><span style="display:flex;"><span>-c \
</span></span><span style="display:flex;"><span>--max_loci_files 500
</span></span></code></pre></div>
<div class="box notices cstyle warning">
    <div class="box-label"><i class="fa-fw fas fa-exclamation-triangle"></i> Warning</div>
    <div class="box-content">
<p>Notice the addition of option <code>--max_loci_files 500</code>, which is used only for showing the output directories containing separate FASTA file per marker (<strong>3&rsquo;</strong>, <strong>4&rsquo;</strong>, <strong>5&rsquo;</strong>, <strong>6&rsquo;</strong>, <strong>13&rsquo;</strong>, and <strong>14&rsquo;</strong> in the image below, not created by default), we don&rsquo;t recommend using this option in your runs since it will unnecesarily create large numbers of small FASTA files which would have to be concatenated again anyways during the alignment step.<br>
You can read more about the option here: <a href="/captus.docs/assembly/extract/options/#--max_loci_files"><strong>&ndash;max_loci_files</strong></a></p>
    </div>
</div>
<p>After the run is finished we should see a new directory called <code>03_extractions</code> with the following structure and files:</p>
<p><img src="/captus.docs/images/extractions.png?width=640&amp;classes=shadow" alt="Extractions"></p>
<h3 id="1-sample_name__captus-ext">1. <strong><code>[SAMPLE_NAME]__captus-ext</code></strong></h3>
<p>A subdirectory ending in <code>__captus-ext</code> is created to contain the extracted markers of each sample separately (<strong>S1</strong>, <strong>S2</strong>, <strong>S3</strong>, and <strong>S4</strong> in the image).</p>
<hr>
<h3 id="2-01_coding_nuc-02_coding_ptd-03_coding_mit">2. <strong><code>01_coding_NUC</code></strong>, <strong><code>02_coding_PTD</code></strong>, <strong><code>03_coding_MIT</code></strong></h3>
<p>These directories contain the extracted <strong>coding</strong> markers from the <strong>NUC</strong>lear, <strong>P</strong>las<strong>T</strong>i<strong>D</strong>ial, and <strong>MIT</strong>ochondrial genomes respectively.<br>
The markers are presented in four formats: protein sequence (<strong>coding_AA</strong>), coding sequence in nucleotide (<strong>coding_NT</strong>), exons and introns concatenated (<strong>genes</strong>), and the concatenation of exons and introns flanked by a fixed length of sequence (<strong>genes_flanked</strong>):</p>
<p><img src="/captus.docs/images/protein_extraction.png?width=600&amp;classes=shadow" alt="Protein extraction formats"></p>
<hr>
<h3 id="3-marker_type_coding_aafaa-01_aa">3. <strong><code>[MARKER_TYPE]_coding_AA.faa</code></strong>, <strong><code>01_AA</code></strong></h3>
<p>Coding sequence in <strong>aminoacids</strong>. Prefixes can be <code>NUC</code>, <code>PTD</code>, or <code>MIT</code>. For details on sequence headers see <a href="/captus.docs/assembly/extract/output/#fasta-headers-explanation">FASTA headers explanation</a>.</p>

<div class="expand">
    <a class="expand-label" onclick="$t=$(this); if($t.parent('.expand-expanded.expand-marked').length){ $t.next().css('display','none') }else if($t.parent('.expand-marked').length){ $t.next().css('display','block') }else{ $t.next('.expand-content').slideToggle(100); } $t.parent().toggleClass('expand-expanded');">
        <i class="fas fa-chevron-down"></i>
        <i class="fas fa-chevron-right"></i>
        Example
    </a>
    <div class="expand-content" style="display: none;">
<p><img src="/captus.docs/images/coding_AA.png?width=1000&amp;classes=shadow" alt="AA format"></p>

    </div>
</div>
<hr>
<h3 id="4-marker_type_coding_ntfna-02_nt">4. <strong><code>[MARKER_TYPE]_coding_NT.fna</code></strong>, <strong><code>02_NT</code></strong></h3>
<p>Coding sequence in <strong>nucleotides</strong>, a.k.a. CDS. Prefixes can be <code>NUC</code>, <code>PTD</code>, or <code>MIT</code>. For details on sequence headers see <a href="/captus.docs/assembly/extract/output/#fasta-headers-explanation">FASTA headers explanation</a>.</p>

<div class="expand">
    <a class="expand-label" onclick="$t=$(this); if($t.parent('.expand-expanded.expand-marked').length){ $t.next().css('display','none') }else if($t.parent('.expand-marked').length){ $t.next().css('display','block') }else{ $t.next('.expand-content').slideToggle(100); } $t.parent().toggleClass('expand-expanded');">
        <i class="fas fa-chevron-down"></i>
        <i class="fas fa-chevron-right"></i>
        Example
    </a>
    <div class="expand-content" style="display: none;">
<p><img src="/captus.docs/images/coding_NT.png?width=1000&amp;classes=shadow" alt="NT format"></p>

    </div>
</div>
<hr>
<h3 id="5-marker_type_genesfna-03_genes">5. <strong><code>[MARKER_TYPE]_genes.fna</code></strong>, <strong><code>03_genes</code></strong></h3>
<p>Gene sequence (exons in capital letters + introns in lowercase letters) in <strong>nucleotides</strong>. A contig connector of 50 <code>n</code> characters is included when the protein match spans more than a single contig. Prefixes can be <code>NUC</code>, <code>PTD</code>, or <code>MIT</code>. For details on sequence headers see <a href="/captus.docs/assembly/extract/output/#fasta-headers-explanation">FASTA headers explanation</a>.</p>

<div class="expand">
    <a class="expand-label" onclick="$t=$(this); if($t.parent('.expand-expanded.expand-marked').length){ $t.next().css('display','none') }else if($t.parent('.expand-marked').length){ $t.next().css('display','block') }else{ $t.next('.expand-content').slideToggle(100); } $t.parent().toggleClass('expand-expanded');">
        <i class="fas fa-chevron-down"></i>
        <i class="fas fa-chevron-right"></i>
        Example
    </a>
    <div class="expand-content" style="display: none;">
<p><img src="/captus.docs/images/genes.png?width=1000&amp;classes=shadow" alt="GE format"></p>

    </div>
</div>
<hr>
<h3 id="6-marker_type_genes_flankedfna-04_genes_flanked">6. <strong><code>[MARKER_TYPE]_genes_flanked.fna</code></strong>, <strong><code>04_genes_flanked</code></strong></h3>
<p>Gene sequence (exons in capital letters + introns in lowercase letters) plus additional flanking sequence in lowercase <strong>nucleotides</strong>. A contig connector of 50 <code>n</code> characters is included when the protein match spans more than a single contig. Prefixes can be <code>NUC</code>, <code>PTD</code>, or <code>MIT</code>. For details on sequence headers see <a href="/captus.docs/assembly/extract/output/#fasta-headers-explanation">FASTA headers explanation</a>.</p>

<div class="expand">
    <a class="expand-label" onclick="$t=$(this); if($t.parent('.expand-expanded.expand-marked').length){ $t.next().css('display','none') }else if($t.parent('.expand-marked').length){ $t.next().css('display','block') }else{ $t.next('.expand-content').slideToggle(100); } $t.parent().toggleClass('expand-expanded');">
        <i class="fas fa-chevron-down"></i>
        <i class="fas fa-chevron-right"></i>
        Example
    </a>
    <div class="expand-content" style="display: none;">
<p><img src="/captus.docs/images/genes_flanked.png?width=1000&amp;classes=shadow" alt="GF format"></p>

    </div>
</div>
<hr>
<h3 id="7-marker_type_contigs_listtxt">7. <strong><code>[MARKER_TYPE]_contigs_list.txt</code></strong></h3>
<p>List of contig names that had protein hits. Prefixes can be <code>NUC</code>, <code>PTD</code>, or <code>MIT</code>.</p>

<div class="expand">
    <a class="expand-label" onclick="$t=$(this); if($t.parent('.expand-expanded.expand-marked').length){ $t.next().css('display','none') }else if($t.parent('.expand-marked').length){ $t.next().css('display','block') }else{ $t.next('.expand-content').slideToggle(100); } $t.parent().toggleClass('expand-expanded');">
        <i class="fas fa-chevron-down"></i>
        <i class="fas fa-chevron-right"></i>
        Example
    </a>
    <div class="expand-content" style="display: none;">
<div class="highlight"><pre tabindex="0" style="color:#f8f8f2;background-color:#272822;-moz-tab-size:4;-o-tab-size:4;tab-size:4;"><code class="language-text" data-lang="text"><span style="display:flex;"><span>NODE_138_length_18304_cov_18.0000_k_175_flag_1
</span></span><span style="display:flex;"><span>NODE_635_length_5337_cov_17.0000_k_175_flag_1
</span></span><span style="display:flex;"><span>NODE_46_length_16959_cov_19.0000_k_175_flag_1
</span></span><span style="display:flex;"><span>NODE_321_length_4728_cov_10.0000_k_175_flag_1
</span></span><span style="display:flex;"><span>NODE_760_length_19021_cov_17.9856_k_175_flag_0
</span></span><span style="display:flex;"><span>NODE_621_length_11511_cov_11.9845_k_175_flag_0
</span></span><span style="display:flex;"><span>NODE_965_length_6331_cov_15.0000_k_175_flag_1
</span></span><span style="display:flex;"><span>NODE_948_length_26295_cov_58.0000_k_175_flag_0
</span></span><span style="display:flex;"><span>NODE_1726_length_1438_cov_9.0000_k_175_flag_1
</span></span><span style="display:flex;"><span>NODE_210_length_2896_cov_9.9471_k_175_flag_0
</span></span><span style="display:flex;"><span>NODE_677_length_10733_cov_14.0000_k_175_flag_1
</span></span><span style="display:flex;"><span>NODE_996_length_2375_cov_11.0000_k_175_flag_1
</span></span><span style="display:flex;"><span>NODE_1837_length_366_cov_3.0000_k_175_flag_1
</span></span><span style="display:flex;"><span>NODE_1647_length_529_cov_7.0000_k_175_flag_1
</span></span><span style="display:flex;"><span>NODE_792_length_4384_cov_17.0000_k_175_flag_1
</span></span><span style="display:flex;"><span>NODE_1378_length_5491_cov_21.0000_k_175_flag_1
</span></span><span style="display:flex;"><span>NODE_602_length_14961_cov_47.0000_k_175_flag_1
</span></span><span style="display:flex;"><span>NODE_1293_length_649_cov_3.0000_k_175_flag_1
</span></span><span style="display:flex;"><span>NODE_949_length_2240_cov_50.0000_k_175_flag_1
</span></span><span style="display:flex;"><span>NODE_751_length_2777_cov_34.7909_k_175_flag_0
</span></span></code></pre></div>
    </div>
</div>
<hr>
<h3 id="8-marker_type_contigsgff">8. <strong><code>[MARKER_TYPE]_contigs.gff</code></strong></h3>
<p>Annotation track in <a href="http://www.ensembl.org/info/website/upload/gff.html">GFF</a> format for protein hits to contigs in assembly. Prefixes can be <code>NUC</code>, <code>PTD</code>, or <code>MIT</code>.</p>
<p>See <a href="/captus.docs/assembly/extract/output/#19-06_assembly_annotated">19. 06_assembly_annotated</a></p>
<hr>
<h3 id="9-marker_type_recovery_statstsv">9. <strong><code>[MARKER_TYPE]_recovery_stats.tsv</code></strong></h3>
<p>Tab-separated-values table with marker recovery statistics, these are concatenated across marker types and samples and summarized in the final <a href="https://edgardomortiz.github.io/captus.docs/assembly/extract/report/">Marker Recovery report</a>. Prefixes can be <code>NUC</code>, <code>PTD</code>, or <code>MIT</code>.</p>
<p>For more information on the table see <a href="/captus.docs/assembly/extract/output/#26-captus-assembly_extractstatstsv">26. captus-assembly_extract.stats.tsv</a></p>
<hr>
<h3 id="10-marker_type_scipio_finallog">10. <strong><code>[MARKER_TYPE]_scipio_final.log</code></strong></h3>
<p>Log of the second Scipio&rsquo;s run, where best references have already been selected (when using multi-sequence per locus references) and only the contigs that had hits durin Scipio&rsquo;s initial run are used. Prefixes can be <code>NUC</code>, <code>PTD</code>, or <code>MIT</code>.</p>
<hr>
<h3 id="11-00_initial_scipio_marker_type">11. <strong><code>00_initial_scipio_[MARKER_TYPE]</code></strong></h3>
<p>Directory for Scipio&rsquo;s initial run results. The directory contains the set of filtered protein references <code>[MARKER_TYPE]_best_proteins.faa</code> (when using multi-sequence per locus references) and the log of Scipio&rsquo;s initial run <code>[MARKER_TYPE]_scipio_initial.log</code>. Suffixes can be <code>NUC</code>, <code>PTD</code>, or <code>MIT</code>.</p>
<hr>
<h3 id="12-04_misc_dna-05_clusters">12. <strong><code>04_misc_DNA</code></strong>, <strong><code>05_clusters</code></strong></h3>
<p>These directories contain the extracted <strong>miscellaneous DNA</strong> markers, either from a <strong>DNA</strong> custom set of references or from the <strong>CL</strong>uste<strong>R</strong>ing resulting from using the option <code>--cluster_leftovers</code>.<br>
The markers are presented in two formats: matching DNA segments (<strong>matches</strong>), and the matched segments including flanks and other intervening segments not present in the reference (<strong>matches_flanked</strong>).</p>
<p><img src="/captus.docs/images/misc_dna_extraction.png?width=600&amp;classes=shadow" alt="Miscellaneous DNA extraction formats"></p>
<hr>
<h3 id="13-marker_type_matchesfna-01_matches">13. <strong><code>[MARKER_TYPE]_matches.fna</code></strong>, <strong><code>01_matches</code></strong></h3>
<p>Matches per miscellaneous DNA marker in <strong>nucleotides</strong>. Prefixes can be <code>DNA</code> or <code>CLR</code>. For details on sequence headers see <a href="/captus.docs/assembly/extract/output/#fasta-headers-explanation">FASTA headers explanation</a>.</p>

<div class="expand">
    <a class="expand-label" onclick="$t=$(this); if($t.parent('.expand-expanded.expand-marked').length){ $t.next().css('display','none') }else if($t.parent('.expand-marked').length){ $t.next().css('display','block') }else{ $t.next('.expand-content').slideToggle(100); } $t.parent().toggleClass('expand-expanded');">
        <i class="fas fa-chevron-down"></i>
        <i class="fas fa-chevron-right"></i>
        Example
    </a>
    <div class="expand-content" style="display: none;">
<p><img src="/captus.docs/images/matches.png?width=1000&amp;classes=shadow" alt="MA format"></p>

    </div>
</div>
<hr>
<h3 id="14-marker_type_matches_flankedfna-02_matches_flanked">14. <strong><code>[MARKER_TYPE]_matches_flanked.fna</code></strong>, <strong><code>02_matches_flanked</code></strong></h3>
<p>Matches plus additional flanking sequence per miscellaneous DNA marker in <strong>nucleotides</strong>. Prefixes can be <code>DNA</code> or <code>CLR</code>. For details on sequence headers see <a href="/captus.docs/assembly/extract/output/#fasta-headers-explanation">FASTA headers explanation</a>.</p>

<div class="expand">
    <a class="expand-label" onclick="$t=$(this); if($t.parent('.expand-expanded.expand-marked').length){ $t.next().css('display','none') }else if($t.parent('.expand-marked').length){ $t.next().css('display','block') }else{ $t.next('.expand-content').slideToggle(100); } $t.parent().toggleClass('expand-expanded');">
        <i class="fas fa-chevron-down"></i>
        <i class="fas fa-chevron-right"></i>
        Example
    </a>
    <div class="expand-content" style="display: none;">
<p><img src="/captus.docs/images/matches_flanked.png?width=1000&amp;classes=shadow" alt="MF format"></p>

    </div>
</div>
<hr>
<h3 id="15-marker_type_contigs_listtxt">15. <strong><code>[MARKER_TYPE]_contigs_list.txt</code></strong></h3>
<p>List of contig names that had miscellaneous DNA marker hits. Prefixes can be <code>DNA</code> or <code>CLR</code>.</p>

<div class="expand">
    <a class="expand-label" onclick="$t=$(this); if($t.parent('.expand-expanded.expand-marked').length){ $t.next().css('display','none') }else if($t.parent('.expand-marked').length){ $t.next().css('display','block') }else{ $t.next('.expand-content').slideToggle(100); } $t.parent().toggleClass('expand-expanded');">
        <i class="fas fa-chevron-down"></i>
        <i class="fas fa-chevron-right"></i>
        Example
    </a>
    <div class="expand-content" style="display: none;">
<div class="highlight"><pre tabindex="0" style="color:#f8f8f2;background-color:#272822;-moz-tab-size:4;-o-tab-size:4;tab-size:4;"><code class="language-text" data-lang="text"><span style="display:flex;"><span>NODE_1858_length_3636_cov_14.0000_k_175_flag_1
</span></span><span style="display:flex;"><span>NODE_2876_length_3179_cov_25.0000_k_175_flag_1
</span></span><span style="display:flex;"><span>NODE_502_length_5771_cov_37.0000_k_175_flag_1
</span></span><span style="display:flex;"><span>NODE_347_length_475_cov_2.0000_k_175_flag_1
</span></span><span style="display:flex;"><span>NODE_1393_length_297_cov_4.0000_k_175_flag_1
</span></span><span style="display:flex;"><span>NODE_3093_length_960_cov_17.0000_k_175_flag_1
</span></span><span style="display:flex;"><span>NODE_3265_length_1041_cov_18.0000_k_175_flag_1
</span></span></code></pre></div>
    </div>
</div>
<hr>
<h3 id="16-marker_type_contigsgff">16. <strong><code>[MARKER_TYPE]_contigs.gff</code></strong></h3>
<p>Annotation track in <a href="http://www.ensembl.org/info/website/upload/gff.html">GFF</a> format for miscellaneous DNA marker hits to contigs in assembly. Prefixes can be <code>DNA</code> or <code>CLR</code>.</p>
<p>See <a href="/captus.docs/assembly/extract/output/#19-06_assembly_annotated">19. 06_assembly_annotated</a></p>
<hr>
<h3 id="17-marker_type_recovery_statstsv">17. <strong><code>[MARKER_TYPE]_recovery_stats.tsv</code></strong></h3>
<p>Tab-separated-values table with marker recovery statistics, these are concatenated across marker types and samples and summarized in the final <a href="https://edgardomortiz.github.io/captus.docs/assembly/extract/report/">Marker Recovery report</a>. Prefixes can be <code>DNA</code> or <code>CLR</code>.</p>
<p>For more information on the table see <a href="/captus.docs/assembly/extract/output/#26-captus-assembly_extractstatstsv">26. captus-assembly_extract.stats.tsv</a></p>
<hr>
<h3 id="18-marker_type_blat_searchlog">18. <strong><code>[MARKER_TYPE]_blat_search.log</code></strong></h3>
<p>Log of BLAT&rsquo;s run. Prefixes can be <code>DNA</code> or <code>CLR</code>.</p>
<hr>
<h3 id="19-06_assembly_annotated">19. <strong><code>06_assembly_annotated</code></strong></h3>
<p>The main outputs of this directory are a FASTA file containing all the contigs that had hits to the reference markers called <code>[SAMPLE_NAME]_hit_contigs.fasta</code> as well as an annotation track for those markers called <code>[SAMPLE_NAME]_hit_contigs.gff</code>. You can visualize the annotations in <code>Geneious</code>, for example, by importing the FASTA file and then dropping the GFF file on top:
<img src="/captus.docs/images/assembly_annotated.png?width=1280&amp;classes=shadow" alt="Assembly annotated"></p>
<hr>
<h3 id="20-sample_name_hit_contigsfasta">20. <strong><code>[SAMPLE_NAME]_hit_contigs.fasta</code></strong></h3>
<p>This file contains the subset of the contigs assembled by <code>MEGAHIT</code> that had hit to the reference markers. See the red rectangles in <a href="/captus.docs/assembly/extract/output/#19-06_assembly_annotated">19. 06_assembly_annotated</a>.</p>

<div class="expand">
    <a class="expand-label" onclick="$t=$(this); if($t.parent('.expand-expanded.expand-marked').length){ $t.next().css('display','none') }else if($t.parent('.expand-marked').length){ $t.next().css('display','block') }else{ $t.next('.expand-content').slideToggle(100); } $t.parent().toggleClass('expand-expanded');">
        <i class="fas fa-chevron-down"></i>
        <i class="fas fa-chevron-right"></i>
        Example
    </a>
    <div class="expand-content" style="display: none;">
<p><img src="/captus.docs/images/fasta_format.png?width=1000&amp;classes=shadow" alt="FASTA format"></p>

    </div>
</div>
<hr>
<h3 id="21-sample_name_hit_contigsgff">21. <strong><code>[SAMPLE_NAME]_hit_contigs.gff</code></strong></h3>
<p>Unified annotation track in <a href="http://www.ensembl.org/info/website/upload/gff.html">GFF</a> format for ALL the marker types found in the assembly&rsquo;s contigs. See the red rectangles in <a href="/captus.docs/assembly/extract/output/#19-06_assembly_annotated">19. 06_assembly_annotated</a>.</p>
<hr>
<h3 id="22-sample_name_recovery_statstsv">22. <strong><code>[SAMPLE_NAME]_recovery_stats.tsv</code></strong></h3>
<p>Unified tab-separated-values table with marker recovery statistics from ALL the marker types found in the sample, these are concatenated across samples and summarized in the final <a href="https://edgardomortiz.github.io/captus.docs/assembly/extract/report/">Marker Recovery report</a>.</p>
<p>For more information on the table see <a href="/captus.docs/assembly/extract/output/#26-captus-assembly_extractstatstsv">26. captus-assembly_extract.stats.tsv</a></p>
<hr>
<h3 id="23-leftover_contigsfastagz">23. <strong><code>leftover_contigs.fasta.gz</code></strong></h3>
<p>This file contains the subset of the contigs assembled by <code>MEGAHIT</code> that had no hit to the reference markers. The file is compressed to save space. These are the contigs that are used for clustering across samples in order to discover additional homologous markers.</p>
<hr>
<h3 id="24-leftover_contigs_after_custeringfastagz">24. <strong><code>leftover_contigs_after_custering.fasta.gz</code></strong></h3>
<p>This file contains the subset of the contigs assembled by <code>MEGAHIT</code> that had no hit to the reference markers or even to the newly discovered markers derived from clusterin. The file is compressed to save space.</p>
<hr>
<h3 id="25-captus-assembly_extractrefsjson">25. <strong><code>captus-assembly_extract.refs.json</code></strong></h3>
<p>This file stores the paths to all the references used for extraction. This file is necessary so the alignment step can correctly add the references to the final alignments to be used as guides.</p>

<div class="expand">
    <a class="expand-label" onclick="$t=$(this); if($t.parent('.expand-expanded.expand-marked').length){ $t.next().css('display','none') }else if($t.parent('.expand-marked').length){ $t.next().css('display','block') }else{ $t.next('.expand-content').slideToggle(100); } $t.parent().toggleClass('expand-expanded');">
        <i class="fas fa-chevron-down"></i>
        <i class="fas fa-chevron-right"></i>
        Example
    </a>
    <div class="expand-content" style="display: none;">
<div class="highlight"><pre tabindex="0" style="color:#f8f8f2;background-color:#272822;-moz-tab-size:4;-o-tab-size:4;tab-size:4;"><code class="language-json" data-lang="json"><span style="display:flex;"><span>{
</span></span><span style="display:flex;"><span>    <span style="color:#f92672">&#34;NUC&#34;</span>: {
</span></span><span style="display:flex;"><span>        <span style="color:#f92672">&#34;AA_path&#34;</span>: <span style="color:#e6db74">&#34;/Users/emortiz/software/GitHub/Captus/data/Angiosperms353.FAA&#34;</span>,
</span></span><span style="display:flex;"><span>        <span style="color:#f92672">&#34;AA_msg&#34;</span>: <span style="color:#e6db74">&#34;Angiosperms353 /Users/emortiz/software/GitHub/Captus/data/Angiosperms353.FAA&#34;</span>,
</span></span><span style="display:flex;"><span>        <span style="color:#f92672">&#34;NT_path&#34;</span>: <span style="color:#e6db74">&#34;/Users/emortiz/software/GitHub/Captus/data/Angiosperms353.FNA&#34;</span>,
</span></span><span style="display:flex;"><span>        <span style="color:#f92672">&#34;NT_msg&#34;</span>: <span style="color:#e6db74">&#34;Angiosperms353 /Users/emortiz/software/GitHub/Captus/data/Angiosperms353.FNA&#34;</span>
</span></span><span style="display:flex;"><span>    },
</span></span><span style="display:flex;"><span>    <span style="color:#f92672">&#34;PTD&#34;</span>: {
</span></span><span style="display:flex;"><span>        <span style="color:#f92672">&#34;AA_path&#34;</span>: <span style="color:#e6db74">&#34;/Users/emortiz/software/GitHub/Captus/data/SeedPlantsPTD.FAA&#34;</span>,
</span></span><span style="display:flex;"><span>        <span style="color:#f92672">&#34;AA_msg&#34;</span>: <span style="color:#e6db74">&#34;SeedPlantsPTD /Users/emortiz/software/GitHub/Captus/data/SeedPlantsPTD.FAA&#34;</span>,
</span></span><span style="display:flex;"><span>        <span style="color:#f92672">&#34;NT_path&#34;</span>: <span style="color:#e6db74">&#34;/Users/emortiz/software/GitHub/Captus/data/SeedPlantsPTD.FNA&#34;</span>,
</span></span><span style="display:flex;"><span>        <span style="color:#f92672">&#34;NT_msg&#34;</span>: <span style="color:#e6db74">&#34;SeedPlantsPTD /Users/emortiz/software/GitHub/Captus/data/SeedPlantsPTD.FNA&#34;</span>
</span></span><span style="display:flex;"><span>    },
</span></span><span style="display:flex;"><span>    <span style="color:#f92672">&#34;MIT&#34;</span>: {
</span></span><span style="display:flex;"><span>        <span style="color:#f92672">&#34;AA_path&#34;</span>: <span style="color:#e6db74">&#34;/Users/emortiz/software/GitHub/Captus/data/SeedPlantsMIT.FAA&#34;</span>,
</span></span><span style="display:flex;"><span>        <span style="color:#f92672">&#34;AA_msg&#34;</span>: <span style="color:#e6db74">&#34;SeedPlantsMIT /Users/emortiz/software/GitHub/Captus/data/SeedPlantsMIT.FAA&#34;</span>,
</span></span><span style="display:flex;"><span>        <span style="color:#f92672">&#34;NT_path&#34;</span>: <span style="color:#e6db74">&#34;/Users/emortiz/software/GitHub/Captus/data/SeedPlantsMIT.FNA&#34;</span>,
</span></span><span style="display:flex;"><span>        <span style="color:#f92672">&#34;NT_msg&#34;</span>: <span style="color:#e6db74">&#34;SeedPlantsMIT /Users/emortiz/software/GitHub/Captus/data/SeedPlantsMIT.FNA&#34;</span>
</span></span><span style="display:flex;"><span>    },
</span></span><span style="display:flex;"><span>    <span style="color:#f92672">&#34;DNA&#34;</span>: {
</span></span><span style="display:flex;"><span>        <span style="color:#f92672">&#34;AA_path&#34;</span>: <span style="color:#66d9ef">null</span>,
</span></span><span style="display:flex;"><span>        <span style="color:#f92672">&#34;AA_msg&#34;</span>: <span style="color:#66d9ef">null</span>,
</span></span><span style="display:flex;"><span>        <span style="color:#f92672">&#34;NT_path&#34;</span>: <span style="color:#e6db74">&#34;/Volumes/Shuttle500G/for_docs_output/nrDNA.fasta&#34;</span>,
</span></span><span style="display:flex;"><span>        <span style="color:#f92672">&#34;NT_msg&#34;</span>: <span style="color:#e6db74">&#34;/Volumes/Shuttle500G/for_docs_output/nrDNA.fasta&#34;</span>
</span></span><span style="display:flex;"><span>    },
</span></span><span style="display:flex;"><span>    <span style="color:#f92672">&#34;CLR&#34;</span>: {
</span></span><span style="display:flex;"><span>        <span style="color:#f92672">&#34;AA_path&#34;</span>: <span style="color:#66d9ef">null</span>,
</span></span><span style="display:flex;"><span>        <span style="color:#f92672">&#34;AA_msg&#34;</span>: <span style="color:#66d9ef">null</span>,
</span></span><span style="display:flex;"><span>        <span style="color:#f92672">&#34;NT_path&#34;</span>: <span style="color:#e6db74">&#34;/Volumes/Shuttle500G/for_docs_output/03_extractions/01_clustering_data/clust_id79.20_cov80.00_captus_clusters_refs.fasta&#34;</span>,
</span></span><span style="display:flex;"><span>        <span style="color:#f92672">&#34;NT_msg&#34;</span>: <span style="color:#e6db74">&#34;/Volumes/Shuttle500G/for_docs_output/03_extractions/01_clustering_data/clust_id79.20_cov80.00_captus_clusters_refs.fasta&#34;</span>
</span></span><span style="display:flex;"><span>    }
</span></span><span style="display:flex;"><span>}
</span></span></code></pre></div>
    </div>
</div>
<hr>
<h3 id="26-captus-assembly_extractstatstsv">26. <strong><code>captus-assembly_extract.stats.tsv</code></strong></h3>
<p>Unified tab-separated-values table with marker recovery statistics from ALL the markers found in ALL the samples, this table is used to create the final <a href="https://edgardomortiz.github.io/captus.docs/assembly/extract/report/">Marker Recovery report</a>. Even though the report is quite useful for visualization you might need to do more complex statistical analysis, this table is the most appropriate output file for such analyses.</p>

<div class="expand">
    <a class="expand-label" onclick="$t=$(this); if($t.parent('.expand-expanded.expand-marked').length){ $t.next().css('display','none') }else if($t.parent('.expand-marked').length){ $t.next().css('display','block') }else{ $t.next('.expand-content').slideToggle(100); } $t.parent().toggleClass('expand-expanded');">
        <i class="fas fa-chevron-down"></i>
        <i class="fas fa-chevron-right"></i>
        Information included in the table
    </a>
    <div class="expand-content" style="display: none;">
<table>
<thead>
<tr>
<th>Column</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td><strong>sample_name</strong></td>
<td>Name of the sample.</td>
</tr>
<tr>
<td><strong>marker_type</strong></td>
<td>Type of marker. Possible values are <code>NUC</code>, <code>PTD</code>, <code>MIT</code>, <code>DNA</code>, or <code>CLR</code>.</td>
</tr>
<tr>
<td><strong>locus</strong></td>
<td>Name of the locus.</td>
</tr>
<tr>
<td><strong>ref_name</strong></td>
<td>Name of the reference selected for the locus. Relevant when the reference contains multiple sequences per locus like in Angiosperms353 for example.</td>
</tr>
<tr>
<td><strong>ref_coords</strong></td>
<td>Match coordinates with respect to the reference, each segment is expressed as <code>[start]-[end]</code>, segments within the same contig are separated by <code>,</code>, and segments in different contigs are separated by <code>;</code>. For example: <code>1-47;48-354,355-449</code> indicates that a contig contained a segment matching reference coordinates <code>1-49</code> and a different contig matched two segments, <code>48-354</code> and <code>355-449</code> respectively.</td>
</tr>
<tr>
<td><strong>ref_type</strong></td>
<td>Whether the reference is an aminoacid (<code>prot</code>) or nucleotide (<code>nucl</code>) sequence.</td>
</tr>
<tr>
<td><strong>ref_len_matched</strong></td>
<td>Number of residues matched in the reference.</td>
</tr>
<tr>
<td><strong>hit</strong></td>
<td>Paralog ranking, <code>00</code> is assigned to the best hit, secondary hits start at <code>01</code>.</td>
</tr>
<tr>
<td><strong>pct_recovered</strong></td>
<td>Percentage of the total length of the reference sequence that was matched.</td>
</tr>
<tr>
<td><strong>pct_identity</strong></td>
<td>Percentage of sequence identity between the hit and the reference sequence.</td>
</tr>
<tr>
<td><strong>score</strong></td>
<td>Inspired by <code>Scipio</code>&rsquo;s score: <code>(matches - mismatches) / reference length</code>.</td>
</tr>
<tr>
<td><strong>wscore</strong></td>
<td>Weighted score. When the reference contains multiple sequences per locus, the best-matching reference is decided after normalizing their recovered length across references in the locus and multiplying that value by their respective <code>score</code>, thus producing the <code>wscore</code>. Finally <code>wscore</code> is also penalized by the number of frameshifts (if the marker is coding) and number of contigs used in the assembly of the hit.</td>
</tr>
<tr>
<td><strong>hit_len</strong></td>
<td>Number of residues matched in the sample&rsquo;s contig(s) plus the length of the flanking sequence.</td>
</tr>
<tr>
<td><strong>cds_len</strong></td>
<td>If <code>ref_type</code> is <code>prot</code> this number represents the number of residues corresponding to coding sequence (i.e. exons). If the <code>ref_type</code> is <code>nucl</code> this field shows <code>NA</code>.</td>
</tr>
<tr>
<td><strong>intron_len</strong></td>
<td>If <code>ref_type</code> is <code>prot</code> this number represents the number of residues corresponding to intervening non-coding sequence segments (i.e. introns). If the <code>ref_type</code> is <code>nucl</code> this field shows <code>NA</code>.</td>
</tr>
<tr>
<td><strong>flanks_len</strong></td>
<td>Number of residues included in the flanking sequence.</td>
</tr>
<tr>
<td><strong>frameshifts</strong></td>
<td>Positions of the corrected frameshifts in the output sequence. If the <code>ref_type</code> is <code>nucl</code> this field shows <code>NA</code>.</td>
</tr>
<tr>
<td><strong>hit_contigs</strong></td>
<td>Number of contigs used to assemble the hit.</td>
</tr>
<tr>
<td><strong>hit_l50</strong></td>
<td>Least number of contigs in the hit that contain 50% of the recovered length.</td>
</tr>
<tr>
<td><strong>hit_l90</strong></td>
<td>Least number of contigs in the hit that contain 90% of the recovered length.</td>
</tr>
<tr>
<td><strong>hit_lg50</strong></td>
<td>Least number of contigs in the hit that contain 50% of the reference locus length.</td>
</tr>
<tr>
<td><strong>hit_lg90</strong></td>
<td>Least number of contigs in the hit that contain 90% of the reference locus length.</td>
</tr>
<tr>
<td><strong>ctg_names</strong></td>
<td>Name of the contigs used in the reconstruction of the hit. Example: <code>NODE_6256_length_619_cov_3.0000_k_169_flag_1;NODE_3991_length_1778_cov_19.0000_k_169_flag_1</code>, for a hit where two contigs were used.</td>
</tr>
<tr>
<td><strong>ctg_strands</strong></td>
<td>Contig strands (<code>+</code> or <code>-</code>) provided in the same order as <code>ctg_names</code>. Example: <code>+;-</code> indicates that the contig <code>NODE_6256_length_619_cov_3.0000_k_169_flag_1</code> was matched in the positive strand while the contig <code>NODE_3991_length_1778_cov_19.0000_k_169_flag_1</code> was matched in the ngeative strand.</td>
</tr>
<tr>
<td><strong>ctg_coords</strong></td>
<td>Match coordinates with respect to the contigs in the sample&rsquo;s assembly. Each segment is expressed as <code>[start]-[end]</code>, segments within the same contig are separated by <code>,</code>, and segments in different contigs are separated by <code>;</code> which are provided in the same order as <code>ctg_names</code> and <code>ctg_strands</code>. Example: <code>303-452;694-1626,301-597</code> indicates that a single segment was matched in contig <code>NODE_6256_length_619_cov_3.0000_k_169_flag_1</code> in the <code>+</code> strand with coordinates <code>303-452</code>, while two segments were matched in contig <code>NODE_3991_length_1778_cov_19.0000_k_169_flag_1</code> in the <code>-</code> strand with coordinates <code>694-1626</code> and <code>301-597</code> respectively.</td>
</tr>
</tbody>
</table>

    </div>
</div>
<hr>
<h3 id="27-captus-assembly_extractreporthtml">27. <strong><code>captus-assembly_extract.report.html</code></strong></h3>
<p>This is the final <a href="https://edgardomortiz.github.io/captus.docs/assembly/extract/report/">Marker Recovery report</a>, summarizing marker extraction statistics across all samples and marker types.</p>
<hr>
<h3 id="28-captus-assembly_extractlog">28. <strong><code>captus-assembly_extract.log</code></strong></h3>
<p>This is the log from <code>Captus</code>, it contains the command used and all the information shown during the run. If the option <code>--show_less</code> was enabled, the log will also contain all the extra detailed information that was hidden during the run.</p>
<hr>
<h3 id="29-clust_id_cov_captus_clusters_refsfasta">29. <strong><code>clust_id##.##_cov##.##_captus_clusters_refs.fasta</code></strong></h3>
<p>This FASTA file contains the cluster representatives that will be searched and extracted across samples (prefix <code>CLR</code>), the loci names are called <code>captus_#</code>. These represent newly discovered homologus markers in contigs that had no hits to other reference proteins or miscellaneous DNA markers.

<div class="expand">
    <a class="expand-label" onclick="$t=$(this); if($t.parent('.expand-expanded.expand-marked').length){ $t.next().css('display','none') }else if($t.parent('.expand-marked').length){ $t.next().css('display','block') }else{ $t.next('.expand-content').slideToggle(100); } $t.parent().toggleClass('expand-expanded');">
        <i class="fas fa-chevron-down"></i>
        <i class="fas fa-chevron-right"></i>
        Example
    </a>
    <div class="expand-content" style="display: none;">
<div class="highlight"><pre tabindex="0" style="color:#f8f8f2;background-color:#272822;-moz-tab-size:4;-o-tab-size:4;tab-size:4;"><code class="language-text" data-lang="text"><span style="display:flex;"><span>&gt;GenusA_speciesA_CAP-captus_1
</span></span><span style="display:flex;"><span>GACTTGAGCCCCAAAACTAGGTTGGGTGCAGGGGGTCGATCTTGATTTTATTACTCAGGGTGCTTCAGATCAGGTTCTTGCAGCTGAACATGCTTCGGGACATCGACCCTATGGTCAGAATCTTCAATCTGGAAGCTCTGCTGGTGCATCAAGCCAACAAGACATGTCCAAGATCATATGCCAGTAAGAAAACAAGGTATGTAAATACCTCTGCGGGTGGTGCTTTACCTGGAACAGCGTCTGAATCGACTGGCGATATCCCTGCTACTGATGATACCCCCCTGATGGATGCTGCAGGGTAAACAGGTTAATATAGATCTGCTATCTCGAGCCCTGTAACCTGGCTATGCTCTCTTTGCGTCTGCTACTAGATATGCTTCTGGCTTTGCCATTTCTATGAATTCTTATAGAATTTTGATTCAGTCACTATGCAAATTTTATTTGTTATGTTTATAAGGTCTCTTGATTGCGCTCGGGCTCCTGTCTCGGGGGAGCCCTGCGCTCCCTACGCTTACTAGAATATTCATTCTCCCCTATTAAAATAAAAATTTATCGAGATAAAAAAAGAA
</span></span><span style="display:flex;"><span>&gt;GenusA_speciesA_CAP-captus_2
</span></span><span style="display:flex;"><span>ACCAGATTCCTCCATTGTACAGACAACCATAGGGTCCGACTTTGCAAGACGTTTAAGGCCTTCCACAAGCTTGGGAAGGTCAAATGCCACCTTGCACTGGACAGCCACACGTACCACAGGAGATACAGAAAATTTCATGGCTCGAATTGGATGGGCATCAACTTCCTTCTCGTTTGTTAGAGTGGCATTCTTGGTGATAAATTGGTCCAACCCAACCATGGCAACAGTATTACCACAAGGCACATCCTCCACCGTCTCTTGCCTCTTCCCCATCCAAATAACAGTTCTCTGGACACTCTTCACATACAGGTCTTTCTTCTCTCCAGGAACATAGTTTGGACCCATGATTCTAACTTTCAAACCGGTTGAAACTTTCCCAGAAAAGACTCGACCAAAGGCAAAGAACCTGCCCTTGTCTGATGCAGGAATCATCTTTGAGACATAGAGCATAAGAGGTCCCTCAGGGTCACAATTTCTAATGGCACTAGCATATGCGTCATCTAGTGGACCCTCATACAAATTCTCAACACGAT
</span></span><span style="display:flex;"><span>&gt;GenusB_speciesB_CAP-captus_3
</span></span><span style="display:flex;"><span>TCCATACATATCAGAATCAGCATCCTTTTTAGGTCTGTATAGGGTTGAAAGAGTGGGCTGAGCAGTAAATAAACCCTTGTCGTATATGTTGTACTGGTCGTCGTTGGCAAATCCAGAGTCCATTCCTTTATCTTGGTTAAATAGCCTCTGGTCATACATAACCTCTCCCTCTCTACCTGCTCCTGTAGAAGCCATGCCAAGTGCAACCTTTTCACTGATATCACGATCTCTGTCTCTTGTGATCTTACTCTTCTTTCCCATTGCAGCATCTTTAGCTTCCAACCTTCTTTCCCTTTCCCTCTCTCGACGTCGTTCTTCACGAATCTTCTCCCTTTGCAACCGCTCTTCCCTTTCCTCCCTTGTCTCCTTTGGTAAATCCTTCTCTTTTTCTCTTACACGCTCAAATTCTCCTTTCTTCTCAGAAGTATCTACTGTGTTTCTATCAGATGGATAAAGAACTGAAGAAGGCGGTGCAGCTCCTGTTCTCTCAGACCGGGCTTTCTGCGCTAATGCTCGGAGCTCCAGCTCCTTCTTCTCCTTCTGCTTCATAAGCATTTCTTTCTGAACCTTGGATCTCATTGCAACTGCTTCTCTTGCTTTCTGTTCTGCAACATACAGAGCTTCAGATAGCTTTGCAAAGTTATCGTTAATTTGAACTTCTTGAAGGCCTCTCCCATCAGCTGCAAGGCGCTTGTCAAGTGGGATCGTATAACCTTTTGGATTCTTCCAATTTGAAATACAAGGAGGAATCTTCCAATCCTGCTGATCCTTCACAGTGACAGGACGGGGAGGGGAATGCATAACAGGCACAGGTGGAGACCCCGAAGCTCTTGGAACACGTTTATGCTTGAACTTTGGAGGCTCAAGTGGATCAACTGGCATCTCCACCATTCTAATTATTCTCTCCTTGGCGCCTGAATTAAATGCAGCTGATTGTTGAGATGGCTTATACTTGATAAA
</span></span></code></pre></div>
    </div>
</div></p>
<hr>
<h3 id="fasta-headers-explanation">FASTA headers explanation</h3>

<div class="box notices cstyle info">
    <div class="box-label"><i class="fa-fw fas fa-info-circle"></i> Info</div>
    <div class="box-content">
<p>All the FASTA files produced by <code>Captus</code> containing extracted markers follow the same header style:
<img src="/captus.docs/images/fasta_headers.png?width=1200&amp;classes=shadow" alt="FASTA header">
Paralogs are ranked according to their <code>wscore</code> which, in turn, is calculated from the percent identity (<code>ident</code>) as well as the percent coverage (<code>cover</code>) with respect to the selected reference sequence (<code>query</code>). The best hit is always ranked <code>00</code> and secondary hits start at <code>01</code>. When a single hit is found for a marker (like in locus <code>5859</code> in the image) the ranking <code>00</code> is not included in the sequence name, only when multiple hits are found for a marker (like in locus <code>5865</code> in the image) the ranking <code>00</code> or <code>01</code> is included in the sequence name to make them unique. The description field <code>frameshifts</code> is only present when the output sequence has corrected frameshifts and the numbers indicate their position in the output sequence.<br>
As you can see, <strong>Sample name</strong>, <strong>Locus name</strong>, and <strong>Paralog ranking</strong> are separated by double underscores (<code>__</code>). This is the reason why we don&rsquo;t recommend using <code>__</code> inside your sample names (see <a href="https://edgardomortiz.github.io/captus.docs/assembly/clean/preparation/">sample naming convention</a>)</p>
    </div>
</div>
<hr>
<p>Created by <a href="https://edgardomortiz.github.io/captus.docs/more/credits/#edgardo-m-ortiz">Edgardo M. Ortiz</a> (06.08.2021)<br>
Last modified by <a href="https://edgardomortiz.github.io/captus.docs/more/credits/#edgardo-m-ortiz">Edgardo M. Ortiz</a> (03.06.2022)</p>

            <footer class="footline">
            </footer>
          </article>
        </div>
      </main>
    </div>
    <aside id="sidebar" class="default-animation showVisitedLinks">
      <div id="header-wrapper" class="default-animation">
        <div id="header" class="default-animation">
          <a id="logo" href="/">
            <img src="/captus.docs/images/logo.svg" width="270">
          </a>
        </div>
        <div class="searchbox default-animation">
          <label class="a11y-only" for="search-by">Search</label>
          <i class="fas fa-search"></i>
          <input data-search-input id="search-by" type="search" placeholder="Search...">
          <span data-search-clear=""><i class="fas fa-times"></i></span>
        </div>
        <script>
          var contentLangs=['en'];
        </script>
        <script src="/captus.docs/js/auto-complete.js?1729274442" defer></script>
        <script src="/captus.docs/js/lunr.min.js?1729274442" defer></script>
        <script src="/captus.docs/js/lunr.stemmer.support.min.js?1729274442" defer></script>
        <script src="/captus.docs/js/lunr.multi.min.js?1729274442" defer></script>
        <script src="/captus.docs/js/lunr.en.min.js?1729274442" defer></script>
        <script src="/captus.docs/js/search.js?1729274442" defer></script>
      </div>
      <div id="homelinks" class="default-animation">
        <ul>
          <li>
              <a class="padding" href="/captus.docs/"><i class="fas fa-home"></i> Home</a>
          </li>
        </ul>
      </div>
      <div id="content-wrapper" class="highlightable">
        <ul class="topics collapsible-menu">
          <li data-nav-id="/captus.docs/basics/" class="dd-item"><input type="checkbox" id="section-a6deabe447afe1b2e1405bfc57ca61ff" aria-controls="subsections-a6deabe447afe1b2e1405bfc57ca61ff" class="toggle"><label class="a11y-only toggle" for="section-a6deabe447afe1b2e1405bfc57ca61ff" >Submenu Basics</label><a href="/captus.docs/basics/">Basics<i class="fas fa-check read-icon"></i></a><ul id="subsections-a6deabe447afe1b2e1405bfc57ca61ff">
          <li data-nav-id="/captus.docs/basics/overview/" class="dd-item"><a href="/captus.docs/basics/overview/">Overview<i class="fas fa-check read-icon"></i></a></li>
          <li data-nav-id="/captus.docs/basics/installation/" class="dd-item"><a href="/captus.docs/basics/installation/">Installation<i class="fas fa-check read-icon"></i></a></li>
          <li data-nav-id="/captus.docs/basics/parallelization/" class="dd-item"><a href="/captus.docs/basics/parallelization/">Parallelization<i class="fas fa-check read-icon"></i></a></li></ul></li>
          <li data-nav-id="/captus.docs/assembly/" class="dd-item parent"><input type="checkbox" id="section-911ba432c056dd9608a0f55d402289be" aria-controls="subsections-911ba432c056dd9608a0f55d402289be" class="toggle" checked><label class="a11y-only toggle" for="section-911ba432c056dd9608a0f55d402289be" >Submenu Assembly</label><a href="/captus.docs/assembly/">Assembly<i class="fas fa-check read-icon"></i></a><ul id="subsections-911ba432c056dd9608a0f55d402289be">
          <li data-nav-id="/captus.docs/assembly/clean/" class="dd-item alwaysopen"><input type="checkbox" id="section-a4378026e623f4ffa37b722a2c0d2f01" aria-controls="subsections-a4378026e623f4ffa37b722a2c0d2f01" class="toggle" checked><label class="a11y-only toggle" for="section-a4378026e623f4ffa37b722a2c0d2f01" >Submenu Clean</label><a href="/captus.docs/assembly/clean/"><b>1. </b>Clean<i class="fas fa-check read-icon"></i></a><ul id="subsections-a4378026e623f4ffa37b722a2c0d2f01">
          <li data-nav-id="/captus.docs/assembly/clean/preparation/" class="dd-item"><a href="/captus.docs/assembly/clean/preparation/"><i class="fas fa-clipboard-check"></i> Input Preparation<i class="fas fa-check read-icon"></i></a></li>
          <li data-nav-id="/captus.docs/assembly/clean/options/" class="dd-item"><a href="/captus.docs/assembly/clean/options/"><i class="fas fa-cog"></i> Options<i class="fas fa-check read-icon"></i></a></li>
          <li data-nav-id="/captus.docs/assembly/clean/output/" class="dd-item"><a href="/captus.docs/assembly/clean/output/"><i class="fas fa-dna"></i> Output Files<i class="fas fa-check read-icon"></i></a></li>
          <li data-nav-id="/captus.docs/assembly/clean/report/" class="dd-item"><a href="/captus.docs/assembly/clean/report/"><i class="fas fa-chart-bar"></i> HTML Report<i class="fas fa-check read-icon"></i></a></li></ul></li>
          <li data-nav-id="/captus.docs/assembly/assemble/" class="dd-item alwaysopen"><input type="checkbox" id="section-2bc1af4c52c4e9983bd451f9472d3adb" aria-controls="subsections-2bc1af4c52c4e9983bd451f9472d3adb" class="toggle" checked><label class="a11y-only toggle" for="section-2bc1af4c52c4e9983bd451f9472d3adb" >Submenu Assemble</label><a href="/captus.docs/assembly/assemble/"><b>2. </b>Assemble<i class="fas fa-check read-icon"></i></a><ul id="subsections-2bc1af4c52c4e9983bd451f9472d3adb">
          <li data-nav-id="/captus.docs/assembly/assemble/preparation/" class="dd-item"><a href="/captus.docs/assembly/assemble/preparation/"><i class="fas fa-clipboard-check"></i> Input Preparation<i class="fas fa-check read-icon"></i></a></li>
          <li data-nav-id="/captus.docs/assembly/assemble/options/" class="dd-item"><a href="/captus.docs/assembly/assemble/options/"><i class="fas fa-cog"></i> Options<i class="fas fa-check read-icon"></i></a></li>
          <li data-nav-id="/captus.docs/assembly/assemble/output/" class="dd-item"><a href="/captus.docs/assembly/assemble/output/"><i class="fas fa-dna"></i> Output Files<i class="fas fa-check read-icon"></i></a></li>
          <li data-nav-id="/captus.docs/assembly/assemble/report/" class="dd-item"><a href="/captus.docs/assembly/assemble/report/"><i class="fas fa-chart-bar"></i> HTML Report<i class="fas fa-check read-icon"></i></a></li></ul></li>
          <li data-nav-id="/captus.docs/assembly/extract/" class="dd-item parent alwaysopen"><input type="checkbox" id="section-2d7d2bcbd553e306e5bdc5e08cd1ee2c" aria-controls="subsections-2d7d2bcbd553e306e5bdc5e08cd1ee2c" class="toggle" checked><label class="a11y-only toggle" for="section-2d7d2bcbd553e306e5bdc5e08cd1ee2c" >Submenu Extract</label><a href="/captus.docs/assembly/extract/"><b>3. </b>Extract<i class="fas fa-check read-icon"></i></a><ul id="subsections-2d7d2bcbd553e306e5bdc5e08cd1ee2c">
          <li data-nav-id="/captus.docs/assembly/extract/preparation/" class="dd-item"><a href="/captus.docs/assembly/extract/preparation/"><i class="fas fa-clipboard-check"></i> Input Preparation<i class="fas fa-check read-icon"></i></a></li>
          <li data-nav-id="/captus.docs/assembly/extract/options/" class="dd-item"><a href="/captus.docs/assembly/extract/options/"><i class="fas fa-cog"></i> Options<i class="fas fa-check read-icon"></i></a></li>
          <li data-nav-id="/captus.docs/assembly/extract/output/" class="dd-item active"><a href="/captus.docs/assembly/extract/output/"><i class="fas fa-dna"></i> Output Files<i class="fas fa-check read-icon"></i></a></li>
          <li data-nav-id="/captus.docs/assembly/extract/report/" class="dd-item"><a href="/captus.docs/assembly/extract/report/"><i class="fas fa-chart-bar"></i> HTML Report<i class="fas fa-check read-icon"></i></a></li></ul></li>
          <li data-nav-id="/captus.docs/assembly/align/" class="dd-item alwaysopen"><input type="checkbox" id="section-b7d7c1356d6046dce2612e66a4ad8976" aria-controls="subsections-b7d7c1356d6046dce2612e66a4ad8976" class="toggle" checked><label class="a11y-only toggle" for="section-b7d7c1356d6046dce2612e66a4ad8976" >Submenu Align</label><a href="/captus.docs/assembly/align/"><b>4. </b>Align<i class="fas fa-check read-icon"></i></a><ul id="subsections-b7d7c1356d6046dce2612e66a4ad8976">
          <li data-nav-id="/captus.docs/assembly/align/options/" class="dd-item"><a href="/captus.docs/assembly/align/options/"><i class="fas fa-cog"></i> Options<i class="fas fa-check read-icon"></i></a></li>
          <li data-nav-id="/captus.docs/assembly/align/output/" class="dd-item"><a href="/captus.docs/assembly/align/output/"><i class="fas fa-dna"></i> Output Files<i class="fas fa-check read-icon"></i></a></li>
          <li data-nav-id="/captus.docs/assembly/align/report/" class="dd-item"><a href="/captus.docs/assembly/align/report/"><i class="fas fa-chart-bar"></i> HTML Report<i class="fas fa-check read-icon"></i></a></li></ul></li></ul></li>
          <li data-nav-id="/captus.docs/design/" class="dd-item"><a href="/captus.docs/design/">Design<i class="fas fa-check read-icon"></i></a></li>
          <li data-nav-id="/captus.docs/tutorials/" class="dd-item"><input type="checkbox" id="section-3ac49b4868ca27fd0182ba18f8675a7a" aria-controls="subsections-3ac49b4868ca27fd0182ba18f8675a7a" class="toggle"><label class="a11y-only toggle" for="section-3ac49b4868ca27fd0182ba18f8675a7a" >Submenu Tutorials</label><a href="/captus.docs/tutorials/">Tutorials<i class="fas fa-check read-icon"></i></a><ul id="subsections-3ac49b4868ca27fd0182ba18f8675a7a">
          <li data-nav-id="/captus.docs/tutorials/assembly/" class="dd-item alwaysopen"><input type="checkbox" id="section-c43a0d5204ef8e650675143aae3b10bc" aria-controls="subsections-c43a0d5204ef8e650675143aae3b10bc" class="toggle" checked><label class="a11y-only toggle" for="section-c43a0d5204ef8e650675143aae3b10bc" >Submenu Assembly</label><a href="/captus.docs/tutorials/assembly/">Assembly<i class="fas fa-check read-icon"></i></a><ul id="subsections-c43a0d5204ef8e650675143aae3b10bc">
          <li data-nav-id="/captus.docs/tutorials/assembly/basic/" class="dd-item"><a href="/captus.docs/tutorials/assembly/basic/"><i class="fas fa-school"></i> Basic<i class="fas fa-check read-icon"></i></a></li>
          <li data-nav-id="/captus.docs/tutorials/assembly/advanced/" class="dd-item"><a href="/captus.docs/tutorials/assembly/advanced/"><i class="fas fa-graduation-cap"></i> Advanced<i class="fas fa-check read-icon"></i></a></li></ul></li></ul></li>
        </ul>
        <div id="shortcuts">
          <div class="nav-title">More</div>
          <ul>
            <li><a class="padding" href="https://github.com/edgardomortiz/Captus"><i class='fab fa-fw fa-github'></i> GitHub repo</a></li>
            <li><a class="padding" href="/captus.docs/more/credits"><i class='fas fa-fw fa-bullhorn'></i> Credits</a></li>
          </ul>
        </div>
        <div class="footermargin footerLangSwitch footerVariantSwitch footerVisitedLinks footerFooter showVisitedLinks showFooter"></div>
        <hr class="default-animation footerLangSwitch footerVariantSwitch footerVisitedLinks footerFooter showVisitedLinks showFooter">
        <div id="prefooter" class="footerLangSwitch footerVariantSwitch footerVisitedLinks showVisitedLinks">
          <ul>
            <li id="select-language-container" class="footerLangSwitch">
              <div class="padding select-container">
                <i class="fas fa-language fa-fw"></i>
                <span>&nbsp;</span>
                <div class="select-style">
                  <label class="a11y-only" for="select-language">Language</label>
                  <select id="select-language" onchange="location = baseUri + this.value;">
                    <option lang="en" id="en" value="/captus.docs/assembly/extract/output/" selected></option>
                  </select>
                </div>
                <div class="select-clear"></div>
              </div>
            </li>
            <li id="select-variant-container" class="footerVariantSwitch">
              <div class="padding select-container">
                <i class="fas fa-paint-brush fa-fw"></i>
                <span>&nbsp;</span>
                <div class="select-style">
                  <label class="a11y-only" for="select-variant">Theme</label>
                  <select id="select-variant" onchange="window.variants && variants.changeVariant( this.value );">
                    <option id="green" value="green" selected>Green</option>
                  </select>
                </div>
                <div class="select-clear"></div>
              </div>
              <script>window.variants && variants.markSelectedVariant();</script>
            </li>
            <li class="footerVisitedLinks showVisitedLinks"><a class="padding" onclick="clearHistory();"><i class="fas fa-history fa-fw"></i> Clear History</a></li>
          </ul>
        </div>
        <div id="footer" class="footerFooter showFooter"><style>
	#footer {
	  font-size: 13px;
	  height: 100px;
	  margin-left: auto;
	  margin-right: auto;
	  padding: 2rem 1rem;
	  text-align: center;
	  min-width: 230px;
	  max-width: 300px;
	}
	#footer p {
	  margin: 0;
	}
  </style>
  <a class="github-button" href="https://github.com/edgardomortiz/Captus/archive/master.zip" data-icon="octicon-cloud-download" aria-label="Download Captus on GitHub">Download</a>
  <a class="github-button" href="https://github.com/edgardomortiz/Captus" data-icon="octicon-star" data-show-count="true" aria-label="Star Captus on GitHub">Star</a>
  <a class="github-button" href="https://github.com/edgardomortiz/Captus/fork" data-icon="octicon-repo-forked" data-show-count="true" aria-label="Fork Captus on GitHub">Fork</a>
  <p>Built with <a href="https://github.com/McShelby/hugo-theme-relearn"><i class="fas fa-heart"></i></a> by <a href="https://gohugo.io/">Hugo</a></p>
  <script async defer src="https://buttons.github.io/buttons.js?1729274442"></script>

        </div>
      </div>
    </aside>
    <script src="/captus.docs/js/clipboard.min.js?1729274442" defer></script>
    <script src="/captus.docs/js/perfect-scrollbar.min.js?1729274442" defer></script>
    <script src="/captus.docs/js/featherlight.min.js?1729274442" defer></script>
    <script src="/captus.docs/js/theme.js?1729274442" defer></script>
  </body>
</html>
